src/structure_managers/adaptor_strict.hh

```diff
@@ -222,20 +222,17 @@ namespace rascal {
       return this->manager->get_shared_ptr();
     }
 
-    // BUG8486@(till) I deleted the non const getters, because they are not
-    // needed
-    // if this was wrong, please explain
     //! returns the distance between atoms in a given pair
     template <size_t Order, size_t Layer>
     double get_distance(const ClusterRefKey<Order, Layer> & pair) const {
-      return this->distance->operator[](pair);
+      return (*this->distance)[pair];
```

Comment:
what is the reason here, consistency?

Reply:
Mainly that I find it weird to use explicit calls to `operator XXX`, as they are always available with another syntax.

Reply:
I find both syntax are terrible, the operator XX seems a bit more clear to me.

Reply:
ok, I removed this commit